src/scripts/coveralls.sh

```diff
@@ -73,3 +73,10 @@ echo "Reporting coverage"
 set -x
 # shellcheck disable=SC2086
 ./coveralls report $args
+set +x
+
+exit_status=$?
+if [ "${COVERALLS_FAIL_ON_ERROR}" != "1" ]; then
+  exit_status=0
+fi
+exit "${exit_status}"
```

Comment:
I'm wondering whether it would be OK to just add this:

```suggestion
exit 0
```

That's what's done here:

https://github.com/coverallsapp/orb/blob/b48e556b45f4119f82742212b9b75906bbdc9f9e/src/scripts/coveralls.sh#L39